staging/src/k8s.io/client-go/tools/cache/delta_fifo_test.go

```diff
@@ -56,7 +56,10 @@ func (kl literalListerGetter) GetByKey(key string) (interface{}, bool, error) {
 }
 
 func TestDeltaFIFO_basic(t *testing.T) {
-	f := NewDeltaFIFO(testFifoObjectKeyFunc, nil)
+	f := NewDeltaFIFOWithOptions(DeltaFIFOOptions{
+		KeyFunction: testFifoObjectKeyFunc,
+		KnownObjects: nil,
```

Comment:
`KnownObjects` is nil by default if not set

Reply:
Got it, i will update this pr, thank you.